assets/js/util/index.js

```diff
@@ -286,6 +294,9 @@ export const changeToPercent = ( previous, current ) => {
  *
  * @param {Array}  rowData   An array of google charts row data.
  * @param {number} column The column to extract for the sparkline.
+ *
+ * @return {Array} Extracted column of dataset prepared for google charts.
```

Comment:
Capitalize "Google". See above as well (`rowData`).
```suggestion
 * @return {Array} Extracted column of dataset prepared for Google charts.
```